### ISSUE TITLE
Tab issues in chat welcome view

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#268413]
https://github.com/user-attachments/assets/1c1eadfb-e3c9-4ab2-a0df-1d56437d7df5

1. Seems like tab order should be history, plan items, then 'generate instructions'
2. Odd focus outline on the 'generate instructions' link
